go-controller/pkg/libovsdbops/model_client.go

```diff
@@ -4,9 +4,10 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/types"
 	"reflect"
 
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/types"
```

Comment:
I guess changes in `model_client.go` slipped through and are not meant to be included in this commit?

Reply:
Yep, I'll fix this